learning_journal/learning_journal/templates/create.jinja2

```diff
@@ -1,12 +1,12 @@
 {% extends "index.jinja2" %}
 {% block body %}
-    <form action="" method="POST">
+    <form action="{{ request.route_url('create')}}" method="POST">
```

Comment:
One little-known trick is that when you have a form that loaded at a particular url, and you want to submit that form to exactly the same url, you can leave the `action` parameter blank:

```html
<form action="" method="POST">
   ...
</form>
```